README.md

```diff
@@ -508,6 +510,16 @@ Instead, extract operation criteria from the loop and execute DB operation **onc
 
 It may be difficult from an architecture perspective to decouple DB operations from loops. To solve this problem you can use Data Access Class (DAC) pattern to encapsulate all DB operations into some class (or set of classes) and perform DB operations as little as possible (e.g. before loop or after loop using lazy load). Then perform a regular read in a loop, but from internal table encapsulated in DAC, not from DB.
 
+### Prefer JOIN over FAE and RANGE
+
+Join operations are much faster because they don't have a lot of inputs (no additional transfer from Application to DB) and field mapping (ON part of your SELECT) is performed using internal DB structures (no additional conversion needed).
+
+### Using FAE on HANA
```

Comment:
Better to keep practice names in imperative form like "Use FAE on HANA"

But in this case, it doesn't look like a general rule, so something like "Use FAE on HANA to speed up operations" would be perfect